src/data/css-quiz.ts

```diff
@@ -1197,6 +1197,17 @@ const cssQuiz = [
     Explanation:
       "We can use the :last-of-type selector to target an element of a specific type within a container.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/CSS/:last-of-type/"
+  },
+  {
+    Question:
+      "what is syntax of Attribute Selector in CSS?",
+    Answer: "container[attribute="value"]",
```

Comment:
```suggestion
    Answer: "selector[attribute='value']",
```

Reply:
@Ankit-Aslekar 

You have to use single quotes like suggested above to fix the linting errors